prettier.config.ts

```diff
@@ -1,10 +1,9 @@
-import * as prettier from 'prettier'
+import * as prettier from "prettier"
 
 const options: prettier.Options = {
-  proseWrap: 'always',
+  proseWrap: "always",
   semi: false,
-  singleQuote: true,
-  trailingComma: 'none'
+  trailingComma: "none"
```

Comment:
I think this would also be more consistent with go formatting.

```suggestion
  trailingComma: "all",
```

Reply:
Excellent idea. This also minimizes diffs when adding or removing values from arrays.